xml/deployment_yast_installer.xml

```diff
@@ -2447,8 +2447,8 @@ sle-live-patching 8c541494</screen>
    </important>
    <para>
     This category allows hardening your system with &openscap; security
-    policies. The first policy that was implemented is the <literal>&disa;
-     &stiga;</literal>; more policies will be added in future releases.
+    policies. The first policy that was implemented is the
+   <literal>&stig; (&stiga;)</literal> by the <orgname>&disa;</orgname>.
```

Comment:
The reason why I use `&disa; &stiga`; is that resolves to the string in the installer. While I like your change I think we also need introduce the abbreviation `&disaa;`  to use it later (in line 2468).

```suggestion
   <literal>&stig; (&stiga;)</literal> by the <orgname>&disa;</orgname>
   (<orgname>&disaa;</orgname>).
```

Reply:
Good point (did not look at the UI string), makes sense!